src/Commands/SystemCommands/GenericmessageCommand.php

```diff
@@ -46,15 +46,16 @@ public function executeNoDB()
      */
     public function execute()
     {
-        //System command, fetch command to execute if conversation exist
-        $message = $this->getMessage();
-        $chat_id = $message->getChat()->getId();
-        $user_id = $message->getFrom()->getId();
-        //Fetch Conversation if exist
-        $command = (new Conversation($user_id, $chat_id))->getConversationCommand();
-        if (! is_null($command)) {
+        //If a conversation is busy, execute the conversation command after handling the message
```

Comment:
A better word would have been **active**.
Basically, if a conversation is active / busy happening, the command that initiated the conversation should be called.